src/test/java/dk/kb/present/solr/EmbeddedSolrTest.java

```diff
@@ -56,7 +56,7 @@
 public class EmbeddedSolrTest {
 
     private static final Logger log = LoggerFactory.getLogger(EmbeddedSolrTest.class);
-    private static final String solr_home = "src/main/solr";
+    private static final String solr_home = "target/solr";
```

Comment:
here you are defining the path without a trailing slash, and above it is done with it.